sippy-ng/src/component_readiness/RegressedTestsModal.js

```diff
@@ -96,10 +94,11 @@ export default function RegressedTestsModal({
           >
             <Tab label="Unresolved" {...tabProps(0)} />
             <Tab label="Untriaged" {...tabProps(1)} />
-            {triageEntriesExist && <Tab label="Triaged" {...tabProps(2)} />}
-            {!triageEntriesExist && (
-              <Tab label="Triaged Incidents" {...tabProps(2)} />
-            )}
+            <Tab
+              label="Triaged"
+              disabled={!triageEntriesExist}
```

Comment:
This fixes a bug where the "All" tab shows nothing if there are no active triage entries present in the view.